src/OpenApi/Paths.hs

```diff
@@ -127,35 +147,58 @@ data Parameter = Parameter
 instance HasDescription Parameter where
   getDescription = description
 
+parameterRenames :: Map String String
```

Comment:
@mbj I think I've asked this before, but doesn't this declaration here mean that this function will be exported? As I see that the only call site is within the `JSON.FromJSON Parameter` instance, I wonder whether this could be hidden in some way?

Reply:
@snusnu At this point I'm still unsure what is the public API of this library, and I want to avoid the churn on these parts.

Once I consider a hackage release I'll make the exports explicit, for now I'm fine with leakage to make progress on the semantics.

Reply:
@mbj OK, seems fair to me. IIRC most if not all modules in this lib export all their functions, so I agree that a global pass to fix visibility issues is most efficient.